skills/architect-review/SKILL.md

```
@@ -0,0 +1,103 @@
+
+## Convergence Tracking
+
+| Pass | Changes |
+|------|---------|
+```
+
+### 2. Run passes as background agents
+
+Launch each pass using the **Agent tool** with `model: "opus"` and `run_in_background: true`. Wait for each pass to complete before launching the next. Each agent gets this prompt structure:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)